specification/ai/HealthInsights/HealthInsights.RadiologyInsights/tspconfig.yaml

```diff
@@ -0,0 +1,52 @@
+parameters:
+  "service-dir":
+    default: "sdk/healthinsights"
+  "service-name":
+    default: "HealthInsights"
+  "dependencies":
+    "additionalDirectories":
+      - "specification/ai/HealthInsights/HealthInsights.Common/"
+      - "specification/ai/HealthInsights/HealthInsights.OpenAPI/"
+    default: ""
+  "python-sdk-folder":
+    default: "{project-root}/azure-sdk-for-python/"
+  "java-sdk-folder":
+    default: "{project-root}/azure-sdk-for-java/"
+  "js-sdk-folder":
+    default: "{project-root}/azure-sdk-for-js/"
+  "csharp-sdk-folder":
+    default: "{project-root}/azure-sdk-for-net/"
+  "service-directory-name":
+    default: "healthinsights"
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-python":
+    package-pprint-name: '"Azure Health Insights - Radiology Insights"'
+    package-mode: "dataplane"
+    package-version: 1.0.0b1
+    package-dir: "azure-healthinsights-radiologyinsights"
+    package-name: "{package-dir}"
+    emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
+  "@azure-tools/typespec-java":
+    package-dir: "azure-healthinsights-radiologyinsights"
+    namespace: com.azure.health.insights.radiologyinsights
```

Comment:
The error on `package-dir`. It shoudl be `azure-health-insights-radiologyinsights` (see the pattern in namespace)